app/javascript/widget/views/Home.vue

```diff
@@ -34,13 +34,13 @@
         />
       </transition>
     </div>
-    <div v-if="showAttachmentError" class="banner">
+    <div
+      v-if="showBannerMessage"
+      class="banner"
+      :class="`banner banner__${bannerType}`"
+    >
       <span>
-        {{
-          $t('FILE_SIZE_LIMIT', {
-            MAXIMUM_FILE_UPLOAD_SIZE: fileUploadSizeLimit,
-          })
-        }}
+        {{ bannerMessage }}
```

Comment:
Can we make this a separate component? This one and associated bus logic can be moved to a single place.